### PR TITLE
Omr/feature/react zipcode

### PR DESCRIPTION
**Pull request by creaturenex**
added redux toolkit store slice and reducer

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,11 +5,13 @@
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
```
Comment (adamliang0) [inline public/index.html]: Really cool! Quick question: is it possible to use `https://www.npmjs.com/package/bulma` instead importing through a CDN?

**Review by adamliang0 — CHANGES_REQUESTED** [review]
Thank you for taking the time to develop an impressive and beautiful frontend. But while it looks stunning, your decision to import Bulma from a CDN instead of using an npm package necessitates a review before pulling your commit down into the main development branch.

Using a CDN to import Bulma can be convenient for quick prototyping and small projects, it has several drawbacks. For instance, using a CDN may lead to slower page load times, security issues, and potential dependency conflicts with other scripts on the page. Additionally, relying on external resources can make your website vulnerable to unexpected changes and outages.

On the other hand, using an npm package for Bulma provides several benefits, including easier version control, improved security, and the ability to customize and extend the framework to meet our specific needs. Moreover, importing Bulma from a local package ensures that we have full control over the framework and can rely on it consistently across different environments and projects.

Therefore, we kindly request that you reconsider your suggestion to import the library external and opt into using the npm package for Bulma. If you have any further concerns or feedback, please do not hesitate to let us know, and we will be happy to address them.

Thank you for your understanding and cooperation. I have included the relevant link below. 

https://www.npmjs.com/package/bulma

**Comment by creaturenex** [comment]
We only used the cdn for prototyping. We included the cdn script because the initial webpack configuration wasn't done by us. When we ran the dev server, the html page wasn't showing the bulma styling from the included npm package in the package.json. I believe I sent instructions to the team to follow bulma's instruction to incorporate bulma via webpack. We will update the webpack configuration to drop the cdn before requesting the merge. Although we don't expect issues with the config file, be aware issues might arise.

File: public/index.html (hunk 1 above)
Comment (creaturenex) [reply public/index.html]: Honestly dont know bulma can be added multiple ways, cdn is usually reference through script but the cdn might have access to npm package directly.

**Review by adamliang0 — APPROVED** [review]
Thank you for your nice response, your changes are approve. Have a nice weekend!

**Comment by creaturenex** [comment]
thanks

**Pull request merged**